node_cli/main.js

```diff
@@ -0,0 +1,95 @@
+const inquirer = require("inquirer");
+const ora = require("ora");
+
+const tasks = require("./tasks");
+const index = require("./index");
+const watch = require("./watch");
+const users = require("./users");
+
+const choice1 = "Create a task";
+const choice2 = "Show all of my tasks";
+const choice3 = "Get a specific task";
+const choice4 = "Change a task status";
+const choice5 = "Edit a task";
+const choice6 = "Delete a task";
+const choice7 = "Watch for changes";
+const choice8 = "Log out / Quit";
```

Comment:
This has the issue that now you could refer to "choice8" below but its semantic meaning (Log out) could change. More descriptive constant names would get around this: `choiceLogOut`, `choiceWatchForChanges`, etc.

Reply:
One option is to consolidate into a choices enum:

```js
const Choices = {
  CreateTask: "Create a task",
  ShowAllTasks: "Show all of my tasks",
  GetTask: "Get a specific task",
  ChangeTaskStatus: "Change a task status",
  EditTask: "Edit a task",
  DeleteTask: "Delete a task",
  WatchForChanges: "Watch for changes",
  LogOutAndQuit: "Log out / Quit",
}

const answers = await inquirer.prompt({
  type: "rawlist",
  name: "mainMenu",
  message: "What would you like to do?",
  choices: [
    ...Object.values(Choices),
    new inquirer.Separator(),
  ],
});

```